src/module/Ui/templates/layout/default/serlo-home.phtml

```diff
@@ -0,0 +1,203 @@
+                    <ul class="authentification nav navbar-nav">
+                        <li class="dropdown">
+
+                            <a data-toggle="dropdown" class="dropdown-toggle" href="#" aria-expanded="false">
+                                <span class="fa fa-user"></span>
+                            </a>
+
+                            <?php
+                            echo $this->navigation('top_auth_navigation')
+                                ->menu()
+                                ->setUlClass('dropdown-menu dropdown-menu-right')
+                                ->setPartial('layout/navigation/horizontal')
+                                ->render();
+                            ?>
+                        </li>
+                    </ul>
+                    <ul class="notifications nav navbar-nav">
+                        <?php echo $this->notifications()->render(); ?>
+                    </ul>
+                </div>
+            </nav>
+
+            <div id="mobile-nav"></div>
+
+            <?php if (empty($this->placeholder('hideSubjectNav')->getValue())): ?>
+                <nav id="subject-nav">
+                    <div id="subject-nav-wrapper">
+                        <div class="subject-nav-center">
+                            <div class="pull-right controls">
+                                <?php echo $this->placeholder('controls'); ?>
+                            </div>
+                            <ol id="breadcrumbs">
+                            </ol>
+                        </div>
+                        <?php echo $this->partial('layout/partials/search'); ?>
+                    </div>
+                </nav>
+            <?php endif; ?>
+        </div>
+    </header>
+
+    <div id="page">
+        <section class="clearfix">
+            <div id="content-layout" class="clearfix">
+
+                <?php echo $this->partial('layout/partials/notifications'); ?>
+
+                <div class="flasher"></div>
+                <div class="home-header">
+                    <?php echo $this->brand()->getLogo(); ?>
+                    <?php echo $this->brand()->getBrand(); ?>
+                    <div class="header-text"><?php echo $this->translate('The Open Learning Platform'); ?></div>
+                </div>
+
+                <div class="home-serlo-row">
+                    <div class="home-row">
+                        <p>
+                            <?php echo $this->translate(
+                                'We are a grassroots non-profit organization supporting personalized learning and working
+                                towards equal educational opportunities. This learning platform offers thousands of
+                                instructional articles, learning videos and practice exercises for millions of students
+                                worldwide - completely free of charge. Now it’s time to join us in your language.'
```

Comment:
actually, `your language` should be the actual language, but I don't know what to do here in the default...

Reply:
The solution here is fine by me :)